support/logging/types.go

```diff
@@ -38,10 +65,38 @@ type persistence interface {
 	reset()
 }
 
+func env(key, fallback string) string {
+	value := os.Getenv(key)
+	if value == "" {
+		return fallback
+	}
+
+	return value
+}
+
 func GetDefaultConfig() Config {
-	return Config{
+	cfg := Config{
 		Port:        defaultPort,
 		Persistence: defaultPersistence,
 		LogFilename: defaultLogFilename,
+
+		MongoURL: env(envMongoURL, defaultMongoURL),
+
+		MongoUser:           defaultMongoUsername,
+		MongoPass:           defaultMongoPassword,
+		MongoDatabase:       defaultMongoDB,
+		MongoCollection:     defaultMongoCollection,
+		MongoConnectTimeout: defaultMongoConnectTimeout,
+		MongoSocketTimeout:  defaultSocketTimeout,
 	}
+
+	MongoPortStr := env(envMongoPort, strconv.Itoa(defaultMongoPort))
+	MongoPort, err := strconv.Atoi(MongoPortStr)
+	if err == nil {
+		cfg.MongoPort = MongoPort
+	} else {
+		fmt.Println("Could not parse port", MongoPort)
```

Comment:
Please do not print on stdout. If there is an error - maybe this should be returned as an indication to a calling function.